setup.py

```diff
@@ -75,12 +76,18 @@ def build_cmake(self, ext):
 
         # example of cmake args
         config = 'Debug' if self.debug else 'Release'
+        
+        boost_root = ''
+        if 'CONDA_PREFIX' in os.environ and not 'BOOST_ROOT' in os.environ:
```

Comment:
boost has some issues with anaconda. If we're using anaconda and BOOST_ROOT isn't already set, point it at the conda env

Reply:
this looks good, though a slightly simpler way might be to do this after cmake_args is defined, then:
```
if 'CONDA_PREFIX' in os.environ and 'BOOST_ROOT' not in os.environ:
    cmake_args.append('-DBOOST_ROOT={}'.format(os.environ['CONDA_PREFIX']))
```